controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -1709,20 +1709,6 @@
     OperationFailedRuntimeException noChildType(String name);
 
     /**
-     * A message indicating no handler for the step operation, represented by the {@code stepOpName} parameter, at
-     * {@code address}.
-     *
-     * @param stepOpName the step operation name.
-     * @param address    the address.
-     *
-     * @return the message
-     *
-     * @deprecated use {@link #noSuchResourceType(PathAddress)} or {@link #noHandlerForOperation(String, PathAddress)}
-     */
-//    @Message(id = 148, value = "No handler for %s at address %s")
-//    String noHandler(String stepOpName, PathAddress address);
```

Comment:
Please don't do this. The commented code clarifies why there's a gap in the id sequence and makes it easy to use git-annotate to figure out what change made the message unnecessary.